### DISCUSSION TITLE
Is Yetiforce CRM compatible with AlmaLinux?

### DISCUSSION THREAD

**Discussion by BKLTMS** [YetiForceCompany/YetiForceCRM#17429] [General]
Need to upgrade from CentOS 7 to AlmaLinux 8.  Any issues with Yetiforce and AlmaLinux?  Any special issues to be aware of or configuration needs to perform?
Thanks in advance for your input.

**Comment by mariuszkrzaczkowski** [comment]
yes, read the requirements https://doc.yetiforce.com/introduction/requirements/
